iquip/apps/scheduler.py

```diff
@@ -26,31 +26,31 @@ class _ScheduleThread(QThread):
           If a timeout occurs, i.e. the queue is not changed, the "isChanged" is set to False.
 
     Attributes:
+        updatedTime: The last updated time, in the format of time.time().
         ip: The proxy server IP address.
         port: The proxy server PORT number.
-        updatedTime: The last updated time, in the format of time.time().
     """
 
     fetched = pyqtSignal(bool, float, list)
 
     def __init__(
         self,
+        updatedTime: Optional[float],
         ip: str,
         port: int,
-        updatedTime: Optional[float],
-        callback: Callable[[bool, float, Iterable[SubmittedExperimentInfo]], None],
+        callback: Callable[[bool, float, Sequence[SubmittedExperimentInfo]], None],
```

Comment:
Still, `Callable[[..., List[...]], ...]` is better since the `fetched` always gives a `list`.
You can then connect `Callable[[..., Sequence[...]], ...]` as its slot, just like you did now.

Reply:
Then, is it better to modify `updateScheduleModel()` type hint as well?